src/main/api/2/studio-api-2.yaml

```diff
@@ -4323,6 +4726,251 @@ components:
         - environment
         - items
 
+    SidebarItem:
+      type: object
+      properties:
+        id:
+          type: string
+          description: content item ID
+        label:
+          type: string
+          description: user friendly display name for content item
+        contentTypeId:
+          type: string
+          description: content type of content item
+        path:
+          type: string
+          description: path where content item is stored inside repository (with filename)
+        previewUrl:
+          type: string
+          description: url for previewing content item
+        systemType:
+          type: string
+          description: one of predifined type values (page, component, asset, level descriptor, folder ...)
```

Comment:
We should change this to be the actual strings to expect back as they would eventually be.

Reply:
Please fix.